client/src/components/UIElements/Search.scss

```diff
@@ -0,0 +1,24 @@
+.search-container {
+    position: fixed;
+    bottom: 0;
+    width: 370px;
+}
```

Comment:
super specific to your screen size. please replace with responsive layout.
![image](https://github.com/assafnoahkoren/quiz-app/assets/16115480/0dba2ca0-f9d5-4811-b73e-569b9371b717)